highlight/highlight
# Create TagSwitchGroup component

## Summary

Adds a `TagSwitchGroup` component based on [the mocks in Figma](https://www.figma.com/file/pAHlb8IFmVSewPJcphbjIH/Component-Library?type=design&node-id=3284%3A33009&mode=design&t=noRldW6uxHS4sa1o-1). Also upgrades Storybook packages and updates our config + stories to fix the Storybook build and get docs working after https://github.com/highlight/highlight/pull/8015, which upgraded Storybook but not the supporting packages.

<img width="980" alt="Screenshot 2024-03-27 at 10 21 09 AM" src="https://github.com/highlight/highlight/assets/308182/828835fb-2e45-43c3-96b8-4641db298815">

## How did you test this change?

* Run `yarn dev:storybook` from the `packages/ui` directory
* Confirm behavior of `TagSwitchGroup` works as expected in component stories

## Are there any deployment considerations?

N/A

## Does this work require review from our design team?

Will let Julian know the component is built out, but should not require any additional input.
